core/modules/flowgen.h

```diff
@@ -89,7 +89,7 @@ class FlowGen final : public Module {
   int allocated_flows_;
   uint64_t generated_flows_;
   struct flow *flows_;
-  std::deque<struct flow *> flows_free_;
+  std::stack<struct flow *> flows_free_;
```

Comment:
Could you add a member variable comment for this (and if you feel like it, also for the other members)?